Source/ReactiveCore/MoyaError.swift

```diff
@@ -10,11 +20,23 @@ public enum MoyaErrorCode: Int {
     case Data
 }
 
-public enum MoyaError: ErrorType {
-    case ImageMapping(MoyaResponse)
-    case JSONMapping(MoyaResponse)
-    case StringMapping(MoyaResponse)
-    case StatusCode(MoyaResponse)
-    case Data(MoyaResponse)
-    case Underlying(ErrorType)
+internal extension MoyaError {
+    
+    // Used to convert MoyaError to NSError for RACSignal
+    var nsError: NSError {
```

Comment:
I think the Cocoa convention would make this `NSError` – I know it looks weird, but I think it makes sense to adhere.